### PR TITLE
Add mariner 1.0 image

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Add mariner 1.0 image

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/preview/mariner1/test-deps/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    # provides ps
+    procps-ng \
+    # upgrade packages
+    && yum upgrade -y \
+    # clean cached data
+    && yum clean all \
+    # remove cache folders and files
+    && rm -rf /var/cache/yum
+
+ENV POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-TestDeps-Ubuntu-20.04
```
Comment (RDIL) [inline release/preview/mariner1/test-deps/docker/Dockerfile:28]: Leftover or intentional?

File: release/preview/mariner1/test-deps/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Docker image file that describes an Ubuntu image with PowerShell and test dependencies
```
Comment (anamnavi) [inline release/preview/mariner1/test-deps/docker/Dockerfile:1]:
```suggestion
# Docker image file that describes an Mariner image with PowerShell and test dependencies
```

File: release/preview/mariner1/test-deps/docker/Dockerfile (hunk 1 above)
Comment (anamnavi) [inline release/preview/mariner1/test-deps/docker/Dockerfile:28]:
```suggestion
ENV POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-Mariner-1.0
```

**Review by anamnavi — CHANGES_REQUESTED** [review]

File: release/preview/mariner1/test-deps/docker/Dockerfile (hunk 1 above)
Comment (TravisEz13) [reply release/preview/mariner1/test-deps/docker/Dockerfile:28]:
```suggestion
ENV POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-TestDeps-Mariner-1.0
```

**Comment by TravisEz13** [comment]
thanks, @RDIL  and @anamnavi

**Review by anamnavi — APPROVED** [review]

**Pull request merged**